### DISCUSSION TITLE
Cookie "ory_kratos_continuity" sometimes is not attached

### DISCUSSION THREAD

**Discussion by karolis-palsauskas-devoteam** [ory/kratos#4257] [Q&A]
Hello,

We use kratos for identity management, it was done by our infra/backend, I myself work on frontend, never did anything with kratos, but one bug I suspect is coming from Kratos was assigned to me and now I am looking for a little help.

So, I noticed that after successful authentication via Google for example, when it redirects back to our app, we call identity manager endpoint "/whoami" and it fails sometimes with 401. After investigating I see all failed calls had "ory_kratos_continuity" cookie missing. I don't find any info in docs about this particular cookie. 

Has anyone have ideas how to debug? Where to start? Or maybe someone encountered such problem?

**Comment by vinckr** [comment]
Hello @karolis-96 
what version of Ory Kratos are you using? what is the error you see in Kratos, can you provide any logs?
